app/controllers/admin/gdpr_delete_requests_controller.rb

```diff
@@ -3,7 +3,10 @@ class GDPRDeleteRequestsController < Admin::ApplicationController
     layout "admin"
 
     def index
-      @gdpr_delete_requests = ::GDPRDeleteRequest.order(created_at: :desc).page(params[:page]).per(50)
+      @gdpr_delete_requests = Admin::GDPRDeleteRequestsQuery.call(relation: ::GDPRDeleteRequest.all,
```

Comment:
This PR looks great and is 100% shippable.

You would not need to pass the `relation:` as that's defined as a parameter with a default value on the `.call` method.